src/scripts/Field.js

```diff
@@ -0,0 +1,154 @@
+'use strict';
+
+const Cell = require('./Cell');
+
+class Field {
+  constructor(element) {
+    this.cells = [];
+
+    const rows = element.querySelectorAll('.field-row');
+
+    for (let i = 0; i < rows.length; i++) {
+      const row = rows[i].querySelectorAll('.field-cell');
+
+      this.cells.push([...row].map((cell) => new Cell(cell)));
+    }
+  }
+
+  getMaxValue() {
+    let maxValue = 0;
+
+    for (let row = 0; row < this.cells.length; row++) {
+      for (let column = 0; column < this.cells[row].length; column++) {
+        const cellValue = this.cells[row][column].value;
+
+        if (cellValue > maxValue) {
+          maxValue = cellValue;
+        }
+      }
+    }
+
+    return maxValue;
+  }
+
+  hasAvailableMoves() {
+    for (let row = 0; row < this.cells.length; row++) {
```

Comment:
Consider using 'forEach' or 'map' instead of the 'for' loop for better readability and performance. You could use a nested 'forEach' loop to iterate over the 2D 'cells' array.